### PR TITLE
feat: [CCM-8534]: K8s Quick Create

### PR DESCRIPTION
**Pull request by rajarshee-harness**
#### Screenshots

Screenshot 2022-11-02 at 10 35 41 AM

Screenshot 2022-11-02 at 10 30 12 AM

Screenshot 2022-11-02 at 10 30 25 AM

Screenshot 2022-11-02 at 10 30 39 AM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/75-ce/components/K8sQuickCreate/K8sQuickCreateModal.module.scss [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    background-color: var(--blue-50);
+    margin-bottom: var(--spacing-small);
+  }
+
+  .nameInput {
+    width: 320px;
+  }
+
+  .progressBar {
+    background: rgb(232, 232, 249);
```
Comment (jenil-harness) [inline src/modules/75-ce/components/K8sQuickCreate/K8sQuickCreateModal.module.scss:97]: No standard color?

File: src/modules/75-ce/components/K8sQuickCreate/K8sQuickCreateModal.module.scss [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+  .retestBtn {
+    position: absolute;
+    right: 90px;
+    top: 40px;
+  }
+
+  .delegateErrorCtn {
+    padding: var(--spacing-medium) var(--spacing-large);
+    background-color: rgba(252, 237, 236, 0.5);
```
Comment (jenil-harness) [inline src/modules/75-ce/components/K8sQuickCreate/K8sQuickCreateModal.module.scss:32]: No standard color?

File: src/modules/75-ce/pages/cloud-integration/CloudIntegrationPage.module.scss [hunk 3]
```diff
@@ -53,34 +53,87 @@
...
+          border-radius: 4px;
+        }
+
+        grid-template-columns: repeat(4, 100px);
+        grid-template-rows: 100px;
+        column-gap: var(--spacing-xlarge);
+        margin-top: var(--spacing-medium);
+      }
+
+      > div:nth-child(2) {
```
Comment (jenil-harness) [inline src/modules/75-ce/pages/cloud-integration/CloudIntegrationPage.module.scss:121]: No other way to do this?

File: src/modules/75-ce/pages/cloud-integration/CloudIntegrationPage.module.scss [hunk 4]
```diff
@@ -53,34 +53,87 @@
...
+    margin-top: var(--spacing-medium);
 
-    > div:nth-child(1) {
-      :global(.bp3-card) {
-        display: flex;
-        justify-content: center;
-        align-items: center;
-        border-radius: 4px;
-      }
+    :global(.bp3-card) {
```
Comment (jenil-harness) [inline src/modules/75-ce/pages/cloud-integration/CloudIntegrationPage.module.scss:67]: bp3 card must be taking classname as props to override this

File: src/modules/75-ce/components/K8sQuickCreate/steps/StepContainer.tsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    return (
+      <Container className={css.stepCtn}>
+        <Text font={{ variation: FontVariation.TABLE_HEADERS }} color={Color.PRIMARY_7} className={css.stepNumber}>
+          {`${getString('step')} ${stepId}`}
+        </Text>
+        <Container>{children}</Container>
+      </Container>
+    )
+  } else {
+    return <></>
```
Comment (jenil-harness) [inline src/modules/75-ce/components/K8sQuickCreate/steps/StepContainer.tsx:41]: better to return null.

**Review by jenil-harness — CHANGES_REQUESTED** [review]

File: src/modules/75-ce/components/K8sQuickCreate/K8sQuickCreateModal.module.scss (hunk 1 above)
Comment (rajarshee-harness) [reply src/modules/75-ce/components/K8sQuickCreate/K8sQuickCreateModal.module.scss:97]: Done.

Changed it to nearest standard color.

File: src/modules/75-ce/components/K8sQuickCreate/K8sQuickCreateModal.module.scss (hunk 2 above)
Comment (rajarshee-harness) [reply src/modules/75-ce/components/K8sQuickCreate/K8sQuickCreateModal.module.scss:32]: The color was `Color.RED_50` with 0.5 opacity, so had to convert it to RGBA.

File: src/modules/75-ce/pages/cloud-integration/CloudIntegrationPage.module.scss (hunk 3 above)
Comment (rajarshee-harness) [reply src/modules/75-ce/pages/cloud-integration/CloudIntegrationPage.module.scss:121]: I am reusing an existing component with similar functionality, so had to use this way to override it's styles.

File: src/modules/75-ce/pages/cloud-integration/CloudIntegrationPage.module.scss (hunk 4 above)
Comment (rajarshee-harness) [reply src/modules/75-ce/pages/cloud-integration/CloudIntegrationPage.module.scss:67]: Done. 

Passed a className to `Card`

File: src/modules/75-ce/components/K8sQuickCreate/steps/StepContainer.tsx (hunk 5 above)
Comment (rajarshee-harness) [reply src/modules/75-ce/components/K8sQuickCreate/steps/StepContainer.tsx:41]: Done

**Review by jenil-harness — APPROVED** [review]

**Review by jyotiarora-harness — APPROVED** [review]

**Pull request merged**